### PR TITLE
[CLOUD-3331] - OpenShift Images for 7.3 Beta

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-3331
Signed-off-by: Ken Wills <kwills@redhat.com>

### CODE REVIEW

File: jboss-eap-config-openshift/EAP7.3.0.Beta/added/standalone-openshift.xml [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+            </mail-session>
+        </subsystem>
+        <subsystem xmlns="urn:jboss:domain:messaging-activemq:7.0">
+            <!-- ##MESSAGING_SUBSYSTEM_CONFIG## -->
+        </subsystem>
+        <subsystem xmlns="urn:wildfly:microprofile-config-smallrye:1.0">
+            <!-- ##MICROPROFILE_CONFIG_SOURCE## -->
+        </subsystem>
+        <subsystem xmlns="urn:wildfly:microprofile-health-smallrye:1.0"/>
+        <subsystem xmlns="urn:wildfly:microprofile-metrics-smallrye:2.0" security-enabled="false" exposed-subsystems="*" prefix="${wildfly.metrics.prefix:wildfly}"/>
```
Comment (bstansberry) [inline jboss-eap-config-openshift/EAP7.3.0.Beta/added/standalone-openshift.xml]: In CD 17 this is ${wildfly.metrics.prefix:jboss}"

File: os-eap-launch/added/launch/datasource-common.sh [hunk 2]
```diff
@@ -514,9 +514,6 @@ function inject_datasource() {
       checker="org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLValidConnectionChecker"
       sorter="org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLExceptionSorter"
       ;;
-    "MONGODB")
-      continue
-      ;;
```
Comment (bstansberry) [inline os-eap-launch/added/launch/datasource-common.sh]: Is there any issue related to this?

File: jboss-eap-config-openshift/EAP7.3.0.Beta/added/standalone-openshift.xml (hunk 1 above)
Comment (luck3y) [reply jboss-eap-config-openshift/EAP7.3.0.Beta/added/standalone-openshift.xml]: OK -- I matched what was in the .zip standalone-full-ha.xml, is that an issue?

File: os-eap-launch/added/launch/datasource-common.sh (hunk 2 above)
Comment (luck3y) [reply os-eap-launch/added/launch/datasource-common.sh]: No, its just a leftover, I can create one, but it should have been dropped with the PR that removed mongo originally.

File: jboss-eap-config-openshift/EAP7.3.0.Beta/added/standalone-openshift.xml (hunk 1 above)
Comment (luck3y) [reply jboss-eap-config-openshift/EAP7.3.0.Beta/added/standalone-openshift.xml]: Changed to: ${wildfly.metrics.prefix:jboss}

(I'll squash before you finally approve if that works).

Comment (luck3y) [reply jboss-eap-config-openshift/EAP7.3.0.Beta/added/standalone-openshift.xml]: Well, maybe I'll just squash now :)

File: os-eap-launch/added/launch/datasource-common.sh (hunk 2 above)
Comment (bstansberry) [reply os-eap-launch/added/launch/datasource-common.sh]: That's fine as is.

**Pull request merged**